packages/main/src/RPA/Browser/Selenium.py

```diff
@@ -1696,6 +1698,68 @@ def clear_all_highlights(self):
         )
         self.driver.execute_script(script, *elements)
 
+    @keyword
+    def print_to_pdf(
+        self, source: str = None, output_path: str = None, params: dict = None
+    ):
+        """
+        Print the page to a PDF document using Chromium devtools.
+
+        By default it prints the currently open page.
+
+        For Chrome print parameters see
+        https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-printToPDF
+
+        ``source`` Page to print, by default the current page.
+        ``output_path`` filepath for the generated pdf. By default it is saved to
+          the output folder with name `out.pdf`.
+        ``params`` parameters for the Chrome print method. By default uses values
+            {
+                "landscape": False,
+                "displayHeaderFooter": False,
+                "printBackground": True,
+                "preferCSSPageSize": True,
+            }
+        """
+        if "chrom" not in self.driver.name:
+            self.logger.warning("PDF printing works only with Chrome/Chromium")
+            return
```

Comment:
If this can't actually do the thing it promises, should it just throw an exception?